plumbing/mpool/pool.go

```diff
@@ -0,0 +1,49 @@
+package mpool
+
+import (
+	"context"
+
+	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	"gx/ipfs/QmVRxA4J3UPQpw74dLrQ6NJkfysCA1H4GU28gVpXQt9zMU/go-libp2p-pubsub"
+
+	"github.com/filecoin-project/go-filecoin/core"
+	"github.com/filecoin-project/go-filecoin/plumbing/msg"
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+// Pool interfaces to the core message pool
+type Pool struct {
```

Comment:
mpool.pool stutters pretty bad. we could actually stick this in the msg package and have it be msg.pool. alternatively, remove could just be implemented by having the plumbing api call directly into the core.messagepool. then this could be an mpl.lser or mpl.viewer.

Reply:
Gone